### ISSUE TITLE
Allow adding books as an edition when editing their entry

### ISSUE THREAD

**Issue by hartsick** [bookwyrm-social/bookwyrm#562]
**Is your feature request related to a problem? Please describe.**
I recently read The History of Standard Oil and logged the main search result. After finishing the book in BookWyrm, I found that the [version of the book I was actually reading](https://bookwyrm.social/book/92639) was available from Open Library, but it wasn't considered one of the [editions](https://bookwyrm.social/book/8284/editions) of the book I logged. I would like to edit the book's entry so that I can switch to having logged the Belt Publishing version I read.

**Describe the solution you'd like**
When editing [a book entry](https://bookwyrm.social/book/92639), I would like to be able to associate it to [an edition](https://bookwyrm.social/book/8284/editions). Not sure what the user interaction for this should look like.

**Describe alternatives you've considered**

**Additional context**

**Comment by mouse-reeve** [comment]
thank you yes! the technical solution here would be merging duplicate works

**Comment by jaschaurbach** [comment]
I want to suggest to import more identifiers from OpenLibrary like the ID of the Deutsche Nationalbibliothek (DNB) - every edition published in Germany gets a new ID - no matter if the ISBN has changed or not. I recently had a book where the 1st till 6th edition had the same cover, the 7th had a new one but the same ISBN. Of course I had the 2nd edition and it just looked wrong to see the correct cover for the ISBN but not for the Edition.

**Comment by mouse-reeve** [comment]
Yes, getting DNB would be great. We're already grabbing BNF IDs, and there's no reason to leave Germany out of the party

**Comment by joric3** [comment]
We already have duplicate authors, too - is there already an issue for this, or can it be considered a subset of duplicate books?

**Comment by tinyweasel** [comment]
I think I just added five dupes by pressing "import from x" a lot of times as it had no loading state (sorry). They're not even different editions. Is there functionality to dedupe those? Or have the importer check existing? As there was in fact one or two entries already which search didn't find, hence why I imported.
